scripts/create_service_principal.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+#echo "Usage:
+#  1 bash create-service-principal.sh
+#  2 bash create-service-principal.sh <Subscription ID>
+# You need Azure CLI: https://docs.microsoft.com/azure/xplat-cli-install
+
+SUBSCRIPTION_ID=$1
+
+#echo ""
+#echo "  Background article: https://azure.microsoft.com/documentation/articles/resource-group-authenticate-service-principal"
+echo ""
+
+my_app_name_uuid=$(python -c 'import uuid; print (str(uuid.uuid4())[:8])')
+MY_APP_NAME="app${my_app_name_uuid}"
+
+MY_APP_KEY=$(python -c 'import uuid; print (uuid.uuid4().hex)')
+
+my_app_id_URI="${MY_APP_NAME}_id"
+
+#check if the user has subscriptions. If not she's probably not logged in
+subscriptions_list=$(azure account list --json)
```

Comment:
Can you change it to use az cli ? 
https://github.com/Azure/azure-cli

For this sentense, it is like `az account list --json`